projects/samples/contests/robocup/controllers/referee/tests/launch_all_tests.sh

```diff
@@ -44,22 +44,22 @@ do
     folder=$(dirname ${test_file})
     test_log="${folder}/test.log"
     referee_log="${folder}/referee.log"
-    msg_prefix="[$NTH_TEST/${#TEST_FILES[@]}] $folder"
+    msg_prefix="[$NTH_TEST/${#TEST_FILES[@]}] ${folder:2}"
 
     if should_run_tests
     then
-        echo "$msg_prefix ..."
-        ./launch_test.sh ${folder} &> ${test_log}
+        printf '%-60.60srunning...' "$msg_prefix"
+        ./launch_test.sh ${folder} | tee ${test_log} > /dev/null
```

Comment:
This change has a subtle bug: `tee` only stores stdout in the file - but not stderr. You should use something like `./launch_test.sh ${folder} |& tee ${test_log} > /dev/null` instead.

Reply:
Yes, I was aware of that. But I was not sure, as stderr is not used to log test failures, but rather to log script errors (outside the scope of the test itself). So I had the feeling it would be better to display the stderr in the console rather than redirecting it into the log file. If redirected to the log file, we would not notice the script errors (they would somehow be hidden). Otherwise, we can spot the script errors more easily as they pop-up in the console.